src/js/script.js

```diff
@@ -0,0 +1,270 @@
+/*============================================================================================================= */
```

Comment:
The JS is not properly structured. Individual JS logic should reside along side the component. 

Make below changes: 

Keep your components in a folder where hbs, scss and js file will be. 

eg:
molecules
   - yourComponent
      - yourComponent.hbs
      - yourComponent.scss
      - yourComponent.js